assets/src/components/p4_table/edit.js

```diff
@@ -67,56 +54,51 @@ import {
 
 const BACKGROUND_COLORS = [
 	{
-		color: '#f3f4f5',
-		name: 'Subtle light gray',
-		slug: 'subtle-light-gray',
+		color: '#f5f7f8',
+    name: __('Grey', 'planet4-blocks-backend'),
+    slug: 'grey'
 	},
 	{
-		color: '#e9fbe5',
-		name: 'Subtle pale green',
-		slug: 'subtle-pale-green',
+		color: '#eafee7',
+    name: __('Green', 'planet4-blocks-backend'),
+    slug: 'green'
 	},
 	{
 		color: '#e7f5fe',
-		name: 'Subtle pale blue',
-		slug: 'subtle-pale-blue',
-	},
-	{
-		color: '#fcf0ef',
-		name: 'Subtle pale pink',
-		slug: 'subtle-pale-pink',
-	},
+    name: __('Blue', 'planet4-blocks-backend'),
+    slug: 'blue'
+	}
 ];
 
 const ALIGNMENT_CONTROLS = [
 	{
-		icon: alignLeft,
-		title: __( 'Align column left' ),
+		icon: "editor-alignleft",
+		title: __( 'Align Column Left', 'planet4-blocks-backend' ),
 		align: 'left',
 	},
 	{
-		icon: alignCenter,
-		title: __( 'Align column center' ),
+		icon: "editor-aligncenter",
+		title: __( 'Align Column Center', 'planet4-blocks-backend' ),
 		align: 'center',
 	},
 	{
-		icon: alignRight,
-		title: __( 'Align column right' ),
+		icon: "editor-alignright",
+		title: __( 'Align Column Right', 'planet4-blocks-backend' ),
 		align: 'right',
 	},
 ];
 
 const withCustomBackgroundColors = createCustomColorsHOC( BACKGROUND_COLORS );
 
 const cellAriaLabel = {
-	head: __( 'Header cell text' ),
-	body: __( 'Body cell text' ),
-	foot: __( 'Footer cell text' ),
+	head: __( 'Header cell text', 'planet4-blocks-backend' ),
```

Comment:
Why are we adding our text domain here? :thinking: All these strings are already translated by core, and for blocks where we don't override we also just use the core translations.